src/cli/tc.rs

```diff
@@ -16,22 +15,24 @@ use crate::cli::{self, CLIError};
 use crate::config::reader::ConfigReader;
 use crate::config::Config;
 use crate::print_schema;
+use crate::tracing::default_tracing;
 use crate::valid::Validator;
 
 const FILE_NAME: &str = ".tailcallrc.graphql";
 const YML_FILE_NAME: &str = ".graphqlrc.yml";
 
 pub async fn run() -> Result<()> {
+    let tracing_guard = tracing::subscriber::set_default(default_tracing());
     let cli = Cli::parse();
-    logger_init();
     update_checker::check_for_update().await;
     let runtime = cli::runtime::init(&Upstream::default(), None);
     let config_reader = ConfigReader::init(runtime.clone());
     match cli.command {
         Command::Start { file_paths } => {
             let config_module = config_reader.read_all(&file_paths).await?;
-            log::info!("N + 1: {}", config_module.n_plus_one().len().to_string());
+            tracing::info!("N + 1: {}", config_module.n_plus_one().len().to_string());
             let server = Server::new(config_module);
+            drop(tracing_guard);
```

Comment:
why do we need to drop explicitly ?

Reply:
this is required to disable tracing_subscriber for cli and allow new subscriber that is created based on config to start handling logs